data/transactions/logic/evalStateful_test.go

```diff
@@ -2393,6 +2393,7 @@ func TestReturnTypes(t *testing.T) {
 		// This next one is a cop out.  Can't use itxna Logs until we have inner appl
 		"itxna":         "itxn_begin; int pay; itxn_field TypeEnum; itxn_submit; itxn NumLogs",
 		"base64_decode": `pushbytes "YWJjMTIzIT8kKiYoKSctPUB+"; base64_decode StdEncoding; pushbytes "abc123!?$*&()'-=@~"; ==; pushbytes "YWJjMTIzIT8kKiYoKSctPUB-"; base64_decode URLEncoding; pushbytes "abc123!?$*&()'-=@~"; ==; &&; assert`,
+		"json_ref":      "json_ref JSONInt",
```

Comment:
I'm surprised the test is passing, since I would expect that some json and key bytes need to be pushed to the stack before calling `json_ref`. I wonder if the test is broken.